draft-ietf-teep-architecture.md

```diff
@@ -236,7 +236,14 @@ The following terms are used:
   - Trusted Application (TA): An application component that runs in a TEE.
 
   - Trusted Application (TA) Developer: An entity that wishes to provide functionality
-    on devices that requires the use of one or more Trusted Applications.
+    on devices that requires the use of one or more Trusted Applications. The TA 
+    developer signs the TA binary (or more precisely the manifest 
+    associated with the TA binary) or uses another entity on his or her behalf to get 
```

Comment:
I disagree that the signer necessarily is "used" by the developer.   In other cases, a binary can be countersigned or re-signed by someone the TEEP Agent or TEE trusts.